lib/vdc/NetCDFCollection.cpp

```diff
@@ -491,6 +491,11 @@ bool NetCDFCollection::_GetVariableInfo(string varname, NetCDFSimple::Variable &
     return (true);
 }
 
+NetCDFSimple *NetCDFCollection::newNetCDFSimple() const
```

Comment:
What is the point of replacing "new NetCDFSimple" with newNetCDFSImple()?

Reply:
This is some vestigial code I missed. Removed.